modules/qualitycontrol/CodecSelection.js

```diff
@@ -39,10 +39,16 @@ export class CodecSelection {
         this.visitorCodecs = [];
 
         for (const connectionType of Object.keys(options)) {
-            // eslint-disable-next-line prefer-const
-            let { disabledCodec, preferredCodec, preferenceOrder, screenshareCodec } = options[connectionType];
+            let { disabledCodec, preferredCodec, preferenceOrder } = options[connectionType];
+            const { enableAV1ForFF = false, screenshareCodec } = options[connectionType];
             const supportedCodecs = new Set(this._getSupportedVideoCodecs(connectionType));
 
+            // Adding this flag for testing purposes since AV1 is not properly supported by FF (it doesn't support SVC
+            // and it doesn't send temporal layers with simulcast).
+            if (browser.isFirefox() && !enableAV1ForFF) {
```

Comment:
Can we disable AV1 the way we did it for VP9? Like AV1 decode will be allowed but not encode by pushing it to the end of the list? That way none of the endpoints doing AV1 will have to switch to VP9 when Firefox joins the call.

Reply:
Done